tools/IceRpc.BuildTelemetry.Reporter/README.md

```diff
@@ -0,0 +1,35 @@
+# Build Telemetry
+
+`IceRpc.Protobuf.Tools` and `IceRpc.Slice.Tools` collect anonymous
+build telemetry data about general usage. Participation in this anonymous
+program is optional, and you may [opt-out](#opting-out) if you'd prefer to not
+share any information.
+
+This package includes the `IceRpc.BuildTelemetry.Reporter` client. This client is an
+IceRPC C# client that sends anonymous telemetry data over a secure connection
```

Comment:
And here I think we're overusing 'client'. We say it 3 times in 2 very short sentences.
Personally, I'd get rid of the 2nd one, it's the most redundant here.
```suggestion
This package includes the `IceRpc.BuildTelemetry.Reporter` client. This is an
IceRPC C# client that sends anonymous telemetry data over a secure connection
```